src/test/java/io/phasetwo/service/resource/KeycloakOrganizationExportTest.java

```diff
@@ -0,0 +1,117 @@
+package io.phasetwo.service.resource;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasSize;
+
+import io.phasetwo.client.openapi.model.OrganizationRepresentation;
+import io.phasetwo.service.AbstractOrganizationTest;
+import java.util.List;
+import java.util.Map;
+import lombok.extern.jbosslog.JBossLog;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+@JBossLog
+public class KeycloakOrganizationExportTest extends AbstractOrganizationTest {
```

Comment:
Seems unnecessary to call all of these `KeycloakOrganization...`. Let's drop the `Keycloak`.

Reply:
Done